treeherder/model/models.py

```diff
@@ -279,10 +279,11 @@ def search(cls, search_term):
             # as the ranking algorithm expects english words, not paths
             # So we use standard pattern matching AND trigram similarity to compare suite of characters
             # instead of words
+            # Django already escapes special characters, so we do not need to handle that here
             recent_qs = (
-                Bugscache.objects.filter(summary__icontains=search_term_fulltext)
-                .annotate(similarity=TrigramSimilarity("summary", search_term_fulltext))
-                .order_by("-similarity")[0:max_size]
+                Bugscache.objects.filter(summary__icontains=search_term)
+                .annotate(similarity=TrigramSimilarity("summary", search_term))
+                .order_by("similarity")[0:max_size]
```

Comment:
Should the sorting order not be descending?

Reply:
I initially did that, but that broke several unit tests with mutiple matches

Reply:
@vrigal fixed it in 3b40e92